grouper/fe/handlers/service_account_create.py

```diff
@@ -0,0 +1,112 @@
+from datetime import datetime
+
+from sqlalchemy.exc import IntegrityError
+
+from grouper import group as group_biz
+from grouper.audit import assert_can_join, UserNotAuditor
+from grouper.email_util import send_email
+from grouper.fe.forms import ServiceAccountCreateForm
+from grouper.fe.settings import settings
+from grouper.fe.util import Alert, GrouperHandler
+from grouper.model_soup import (
+        APPROVER_ROLE_INDICIES,
+        Group,
+        GROUP_EDGE_ROLES,
+        GROUP_JOIN_CHOICES,
```

Comment:
Within each grouping, imports should be sorted lexicographically, ignoring case, according to each module's full package path.